queue/group_remote.go

```diff
@@ -16,6 +16,8 @@ import (
 	"gopkg.in/mgo.v2/bson"
 )
 
+const jobsSuffix = ".jobs"
```

Comment:
can we use this constant in the driver implementations that get collection names? and maybe have utility functions to add/strip?

Reply:
done, though in a somewhat different way